include/pagmo/algorithms/compass_search.hpp

```diff
@@ -68,8 +69,7 @@ namespace pagmo
  * a standard in the scientific computing community for exactly the reason observed
  * by Davidon: it is slow but sure'.
  *
- * @note This algorithm does not work for multi-objective problems, nor for
- * constrained optimization or stochastic problems.
+ * @note This algorithm does not work for multi-objective problems, nor for stochastic problems.
```

Comment:
@darioizzo we can do it at a later stage. Just FYI moving forward, not necessary to change it now.